### ISSUE TITLE
Accessibility Feedback for Staging Review | Language Assistance and Resources | C&L Team

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#25592]
# Accessibility Feedback for Staging Review | Language Assistance and Resources | C&L Team

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

**Comment by joshkimux** [comment]
Some issues that may have already been caught from conversations on slack:

## Tagalog page's "On this page" is still in English, not in Tagalog
<img width="715" alt="Screen Shot 2021-06-03 at 10 01 41 AM" src="https://user-images.githubusercontent.com/14154792/120657858-e3043c00-c452-11eb-968e-7b7164defcd4.png">

## Breadcrumb has a `lang="en"` for Tagalog and Spanish content
<img width="1512" alt="Screen Shot 2021-06-03 at 10 03 24 AM" src="https://user-images.githubusercontent.com/14154792/120658011-0c24cc80-c453-11eb-9921-841bd7062693.png">

**Comment by SKasimow** [comment]
@DanielleThierryUSDSVA - can you take a look at comment about "Fine helpful VA phone numbers..." styled as a button

**Comment by DanielleThierryUSDSVA** [comment]
Hi @SKasimow @joshkimux. Sure, yes! That button is a component within the Drupal template for Resources and Support. So I'm looping @johnhashva and @brianalloyd here for further discussion around best solution. John and Brian, please check out Josh's feedback about the link styled as a button on the R&S page. Is this issue being worked on? If not, can we consider for the team's backlog?

**Comment by brianalloyd** [comment]
@DanielleThierryUSDSVA we can definitely add to backlog, I'll do a quick triage to see if we have similar tickets.  Have we also bounced this off the guidance for action links? CC @johnhashva

**Comment by DanielleThierryUSDSVA** [comment]
Awesome. Thank you, @brianalloyd! I don't know that we have re: action links, no. Let me know if you want to chat further about that.

**Comment by SKasimow** [comment]
Hi @brianalloyd - should this be moved to PW's?

**Comment by brianalloyd** [comment]
Probably, I'll move over to our backlog.
